lib/sablon/processor/document.rb

```diff
@@ -117,6 +117,33 @@ def self.encloses?(start_field, end_field)
         end
       end
 
+      class ImageBlock < ParagraphBlock
+        def self.parent(node)
+          node.ancestors
+        end
+
+        def self.encloses?(start_field, end_field)
+          start_field.expression.start_with?('@')
+        end
+
+        def replace(content)
+          if !content.first
+            start_field.remove
+            end_field.remove
+            return
+          end
+
+          pic_prop = self.class.parent(start_field).at_xpath('.//pic:cNvPr', pic: Sablon::Processor::Image::PICTURE_NS_URI)
```

Comment:
As I was looking through the XML produced by the image.docx test case I noticed the id's in `<pic:cNvPr ... >` don't appear to be unique. 

```xml
                        <pic:nvPicPr>
                          <pic:cNvPr id="2" name="1-.jpg"/>
                          <pic:cNvPicPr/>
                        </pic:nvPicPr>
```

```xml
                        <pic:nvPicPr>
                          <pic:cNvPr id="2" name="2-.jpg"/>
                          <pic:cNvPicPr/>
                        </pic:nvPicPr>
```

I had to do some reading but as per this spec I think those ID should be unique to be compliant. Now if they _actually_ do anything or are used I don't know. 
https://msdn.microsoft.com/en-us/library/documentformat.openxml.drawing.pictures.nonvisualdrawingproperties(v=office.14).aspx